src/Chart/Chart.jsx

```diff
@@ -163,12 +182,12 @@ export default () => {
                             }],
                             yAxes: [{
                                 type: 'linear',
-                                min: yRange.max,
-                                max: yRange.min,
+                                min: levelMin,
+                                max: levelMax,
                                 ticks: {
-                                    callback: v => v - yRange.max - yRange.min,
-                                    min: yRange.max,
-                                    max: yRange.min,
+                                    callback: v => v - levelMin - levelMax,
```

Comment:
This looks like a linting issue here, if not, never mind.

Reply:
With our current configuration the linter seems happy with it. 😊